components/classes/content-asset.schema.json

```diff
@@ -0,0 +1,149 @@
+{
+  "meta:license": [
+    "Copyright 2024 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/classes/content-asset",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Content Asset",
+  "type": "object",
+  "meta:extensible": false,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "Content asset such as image or video.",
+  "definitions": {
+    "content-asset": {
+      "properties": {
+        "xdm:assetIdentityID": {
```

Comment:
Do you want to term it as `assetPerceptionID` than `assetIdentityID` ? Identity and ID somehow denote to the same term. There also an `assetID` in the schema.